.github/workflows/pr-golang-check.yaml

```diff
@@ -8,12 +8,17 @@ on:
       - '.github/workflows/pr-golang-check.yaml'
       - 'service/eventlog-server/**.go'
       - 'service/measurement-server/**.go'
+      - 'sdk/golang/ccnp/measurement/**.go'
+      - 'sdk/golang/ccnp/quote/**.go'
```

Comment:
Raised an[ issue](https://github.com/intel/confidential-cloud-native-primitives/issues/142) for here.